packages/ui/package.json

```diff
@@ -12,7 +12,7 @@
     "build:css": "sass --precision=8 src/main.scss ./dist/clr-ui/clr-ui.css; sass --precision=8 src/dark-theme.scss ./dist/clr-ui/clr-ui-dark.css",
     "build:prefix": "postcss ./dist/clr-ui/clr-ui.css -o ./dist/clr-ui/clr-ui.css; postcss ./dist/clr-ui/clr-ui-dark.css -o ./dist/clr-ui/clr-ui-dark.css",
     "build:optimize": "csso -i ./dist/clr-ui/clr-ui.css -o ./dist/clr-ui/clr-ui.min.css -s file --no-restructure; csso -i ./dist/clr-ui/clr-ui-dark.css -o ./dist/clr-ui/clr-ui-dark.min.css -s file --no-restructure",
-    "build:package": "cpy npm.json README.md ./dist/clr-ui/; mv ./dist/clr-ui/npm.json ./dist/clr-ui/package.json; replace '@VERSION' $npm_package_version ./dist/clr-ui/package.json; "
+    "build:package": "cpy npm.json README.md ./dist/clr-ui/; mv ./dist/clr-ui/npm.json ./dist/clr-ui/package.json;"
```

Comment:
Updated to be consistent with the rest of the repo and use the version copy script